docs/OMS-Agent-for-Linux.md

```diff
@@ -380,7 +380,35 @@ If you are using selinux, the semanage tool can be used to allow TCP traffic for
 	  tag oms.syslog
 </source>
 ```
-**Note:** ports 0-1024 are privileged, and require special permissions. As the OMS Agent for Linux runs as the `omsagent` user trying to allocate a source port of 0 to 1024 results in agent startup failure
+### Syslog troubleshooting
+**Note:** ports 0-1024 are privileged, and require special permissions. As the OMS Agent for Linux runs as the `omsagent` user trying to allocate a source port of 0 to 1024 results in agent startup failure.
+
+**Checking for existance of a syslog collector**
+
+Make sure there is a syslog collector available by running the following commands in shell:
+```
+	[ -f /etc/rsyslog.conf ] && echo "rsyslog is configured"
+	[ -f /etc/syslog-ng/syslog-ng.conf ] && echo "syslog-ng is configured"
+```
+
+**Checking port bindings using netstat**
+```
+	PORT=<port you use>
+	netstat -an | grep $PORT
+```
+**Checking port availability using netstat**
+```
+	PORT=<port you want to use>	
+	[ -z "`netstat -an | grep $PORT`" ] && echo "$PORT port is available" || echo "$PORT port is NOT available"
+```
+**Customizing syslog log format using FluentD's format regular expression parameter if syslog data shows up truncated in OMS portal**
+
+OMS agent syslog configuration(see below for the syslog.conf location) is using FluentD's default syslog log format. If your device is generating non-standard syslog formats you can add custom filters to syslog.conf through fluentd 'format' parameter as regexp as documented [here](http://docs.fluentd.org/v0.10/articles/in_syslog).
+**OMS agent syslog configuration file location**
```

Comment:
This line isn't getting indented as I expected; the "OMS agent syslog configuration file location" text shows up right after the "here" link. I think an inline code block would be fine, like so:

OMS agent syslog configuration (in the file `/etc/opt/microsoft/omsagent/<workspace_id>/conf/omsagent.d/syslog.conf`) is using...